projects/question/src/main/java/org/batfish/question/jsonpath/JsonPathQuestionPlugin.java

```diff
@@ -159,7 +162,10 @@ public JsonPathAnswerElement answer() {
                 nodePathResult.setPath(nodesPath);
                 nodePathResult.setNumResults(numResults);
                 boolean includeSuffix = nodesPath.getSuffix();
-                if (!nodesPath.getSummary()) {
+                if (nodesPath.hasValidAssertion()) {
+                  boolean assertion = nodesPath.getAssertion().evaluate(suffixes);
+                  nodePathResult.setAssertionResult(assertion);
+                } else if (!nodesPath.getSummary()) {
```

Comment:
given that the default for JsonPath is to return the prefixes, this seems an odd choice.